pkg/actions/service.go

```diff
@@ -586,6 +579,12 @@ func (s *KVService) PostDeleteBranchHook(_ context.Context, record graveler.Hook
 	s.asyncRun(record)
 }
 
+func (s *KVService) NewRunID() string {
+	entropy := ulid.Monotonic(rand.Reader, 0)
+	tm := time.UnixMilli(unixYear3000 - time.Now().UnixMilli()).UTC()
+	return ulid.MustNew(ulid.Timestamp(tm), entropy).String()
+}
```

Comment:
I think I get the ** unixYear3000** semantics but not the entropy part.
Would you mind explaining everything in a comment? It's so easy to change this later on and break the actions listing.